content/03.results-and-discussion.md

```diff
@@ -4,41 +4,43 @@
+In nf-core/eager, all tools also originally used in EAGER have been updated to latest versions, as available on Bioconda [@doi:10.1038/s41592-018-0046-7] and Conda-forge [@url:https://conda-forge.github.io] to ensure widespread accessibility and stability of utilized tools. The MapDamage2 (for damage profile generation) [@doi:10.1093/bioinformatics/btt193] and Schmutzi for (mitochondrial contamination estimation) [@doi:10.1186/s13059-015-0776-0] methods have not been carried over to nf-core/eager, the first because a more performant successor method (DamageProfiler) exists, and the latter because a stable release of the method could not be migrated to Bioconda. We anticipate that there will be an updated version of Schmutzi in the near future that will allow us to integrate the method again in nf-core/eager, once a version is released on Bioconda. Support for the Bowtie2 aligner [@doi:10.1038/nmeth.1923] will be added in the near future, after consultation with the palaeogenetics community. New tools to the basic workflow include fastp [@doi:10.1093/bioinformatics/bty560] for the removal of 'poly-G' sequencing artefacts that are common in 2-colour Illumina sequencing machines (such as the increasingly popular NextSeq and NovaSeq platforms, [https://sequencing.qcfail.com/articles/illumina-2-colour-chemistry-can-overcall-high-confidence-g-bases/](https://sequencing.qcfail.com/articles/illumina-2-colour-chemistry-can-overcall-high-confidence-g-bases/)). For genotyping we have now included FreeBayes [@doi:1207.3907] as an alternative to the human-focused GATK tools. We have also maintained the possibility of using the now unsupported GATK UnifiedGenotyper, as the GATK HaplotypeCaller performs _de novo_ assembly around possible variants, which may not be suitable for low-coverage aDNA data.
```

Comment:
> the first because a more performant successor method (DamageProfiler) exists

the first because a more performant method is now available (DamageProfiler)